Update activity-1-instructions.Rmd so Details goes to a description (#9)
fhdsl/containers-for-scientists-sandbox: activity-1-instructions.Rmd

Let's point a volume to our workshop files so we have them on our container.

> <details> <summary> Other ways we could get files on a container </summary>

Reviewer: ```suggestion
<details> <summary> Click here for other ways we could get files on a container. </summary>
```

Let's point a volume to our workshop files so we have them on our container.

> <details> <summary> Other ways we could get files on a container </summary>

Reviewer: ```suggestion
<details> <summary> Click here for other ways to get files on a container.</summary>
```
